lib/library_web/controllers/admin_controller.ex

```diff
@@ -38,6 +24,6 @@ defmodule LibraryWeb.AdminController do
     Books.get_book!(id)
     |> Books.delete_book_and_unique_authors
 
-    render(conn, "index.html", books: [])
+    redirect(conn, to: page_path(conn, :index))
```

Comment:
You originally had `books: []` passed in as well, do you still need this?

Reply:
I think index path defaults to displaying all the books